dask/dataframe/tests/test_dataframe.py

```diff
@@ -2813,6 +2813,16 @@ def test_to_timedelta():
               dd.to_timedelta(ds, errors='coerce'))
 
 
+@pytest.mark.skipif(PANDAS_VERSION < '0.22.0',
```

Comment:
@mrocklin, should this use the `hasattr` method of checking for the `isna` instead? Just occurred to me that `isna` could be deprecated in future versions of pandas.

Reply:
I wouldn't worry about isna being deprecated.